config/jobs/kubernetes/test-infra/test-infra-presubmits.yaml

```diff
@@ -18,13 +18,40 @@ presubmits:
         - --config=ci
         - --nobuild_tests_only
         - //...
+        - -//hack:verify-golangci-lint
```

Comment:
I think this needs to be a distinct PR, otherwise the presubmit won't pass here, right? Will it fail if that target doesn't exist yet?

Reply:
Just tested it. I can ignore whole folders but not a single non-existing target. I could open a PR which changes this jobs and adds `verify-golangci-lint` with an empty test-suite. I guess that would work

Reply:
yeah that sounds good, too